hubploy/config.py

```diff
@@ -191,10 +191,10 @@ def get_config(deployment):
             # Multiple images are being built
             images = images_config['images']
 
-    for image in images:
-        # Normalize paths to be absolute paths
-        image['path'] = os.path.join(deployment_path, image['path'])
+        for image in images:
```

Comment:
Slightly confused about the indent changes here too

Reply:
now everything related to images in the `get_config` function is under the conditional statement `if 'images' in config:`

Reply:
Ah, fair. I think the diff was just confusing